lib/mongoose_push/application.ex

```diff
@@ -86,6 +88,39 @@ defmodule MongoosePush.Application do
     end)
   end
 
+  defp metrics do
+    [
+      # Summary is not yet supported in TelemetryMetricsPrometheus
+      Telemetry.Metrics.counter("mongoose_push.push.time",
+        tags: [:status, :service, :type, :reason],
+        tag_values: fn metadata ->
+          case metadata.status do
+            :success ->
+              Map.merge(
+                %{
+                  type: :success,
```

Comment:
`error_category`?

Reply:
In our code I guess the old name is fine without the `error_` prefix, but this will be external name, for someone that doesn't see the code. We need the context.

Reply:
I really don't know what do you mean by `error_category` and where in the code you point out here this is supposed to refer to 😑

Reply:
Most of the functions in this applications return `{:error, {category(), reason()}}` error tuples. This means that error details are actually two atoms, not just one. 
The point I'm trying to make here, is that metadata called `field` attached to a metric that tracks pushed notification is at the very least confusing for reader. Please note, that people reading/operation on metrics have no idea how code looks like and what this `type` refers to. Here, I guess it's supposed to be "error type", so I guess in type above it would be `category()`. But even I knowing the codebase need to think for a moment what this field represents, I can imagine what someone without codebase or even Elixir knowladge would feel. 
This is often what happens, when we decide to "expose" some internal things to outside-world - we need to pay attention to naming, as when this data leaves the service, there is no context given, which renders the naming, very often, incorrcet.